src/main/sensors/acceleration.c

```diff
@@ -89,22 +84,19 @@ void accUpdate(timeUs_t currentTimeUs, rollAndPitchTrims_t *rollAndPitchTrims)
     }
 }
 
-bool accGetAccumulationAverage(float *accumulationAverage)
+float accGetAccumulationAverage(int axis)
```

Comment:
(you can use low-pass filter instead of averaging, with slightly better results)

Reply:
This is for "downsampling" and the logic is the same is in the master branch.
On level modes ATTITUDE/IMU task is called with freq. of 500Hz. ACC task (this one) runs at 1000Hz.

Maybe I can use `acc.accADC` directly because it is already PT2 cutoff=50Hz filtered.

Reply:
Yes, if filter is used, it must run at ACC update frequency (possibly batched). After low-pass filtering, you can decimate samples.
Current implementation does this too, but uses averaging (or boxcar) filter as low-pass. And response of averaging filter is quite poor.

I did not notice that filter is already in place. Applying averaging filter here makes very little sense then (it is working a bit as variable-cutoff filter, with IMU update  frequency as implicit parameter. For 10 point averaging, cutoff frequency is about 40Hz. So previous filter order is increased by one [approximating a lot]),
Using acc.accADC is probably OK for 500Hz sampling, but with 100Hz (Nyquist frequency = 50Hz), it will lead to some frequency aliasing. No big problem in ACRO

Reply:
Updated and it seems to be OK. Nice. Thank you for your ideas.

Reply:
I have updated also Gyro to use PT1. It is already filtered enough.